app/graph/types/asset_type.rb

```diff
@@ -9,6 +9,7 @@ module Types
       field :id, !types.ID, 'Uniq ID for this asset'
       field :asset_type, !types.String, 'type of this Asset'
       field :gemini_token, types.String, 'gemini token for asset'
+      field :image_urls, GraphQL::Types::JSON, 'known image urls'
```

Comment:
One Q I have (and this came up with my `created_at` type) is the use of `GraphQL::Types` versus `types`, scoped within this file -- possible? If so, can we either update all to be `types` or all to be `GraphQL::Types`?

Reply:
Excellent question - I have this ticketed here:

https://artsyproduct.atlassian.net/browse/CSGN-83

I think we're behind other sibling services at Artsy that have already done this migration so I feel like we should be clear to work that one. I might take it one this week!